examples/godot_tps_controller_port/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "tps_controller_port"
+version = "0.1.0"
+edition = "2021"
+authors = ["idanmuze", "The godot-rust developers"]
+publish = false
+rust-version = "1.65"
```

Comment:
Is this actually required? We have a MSRV of 1.63, and while we aren't extremely conservative with bumping it, there should be some kind of a reason.

Reply:
Also, bumping MSRV is a breaking change and would require a new minor version -- even for examples.